runtime/libc/env.c

```diff
@@ -147,3 +145,15 @@ env_getcycles(void)
 
     #endif
 }
+
+// Floating point routines
+// TODO: Do a fair comparison between musl and wasm-musl
+INLINE double
+env_sin(double d) {
+    return sin(d);
+}
+
+INLINE double
+env_cos(double d) {
+    return cos(d);
+}
```

Comment:
This is needed for wasi??

Reply:
I'm not really sure. Where do the env_* functions get used?

These two were previously in libc_backing.c, instead of duplicating them in both wasmception_backing.c and wasi_sdk_backing.c, I moved them into env.c which looks to be where the other env_* functions are located.

Reply:
IIRC these should be put back. They are a hack for wasmception only